src/Catalyst.Abstractions/Catalyst.Abstractions.csproj

```diff
@@ -9,6 +9,9 @@
     <LangVersion>latest</LangVersion>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;CS8002</NoWarn>
```

Comment:
Can you add a bit of explanation with in-line comments on what warnings these are suppressing for future reference, please, Thanks!!

Reply:
I see there are multiple of these in this PR maybe we can put the reason why in a github comment actually then link to that in the file rather than inline!

Reply:
Ok, yeah thesome of those surpress warnings we don't have a lot of control over such as warnings that a project or 3rd party lib doesn't have a strong name key. I'll add some doc.